docker-compose.yml

```diff
@@ -489,7 +489,14 @@ services:
     ulimits: *ulimits
     environment:
       <<: *ccache
+      ARROW_BUILD_STATIC: "OFF"
       ARROW_CUDA: "ON"
+      ARROW_GANDIVA: "OFF"
```

Comment:
I assume this is to reduce build times and focus this on CUDA? 👍 Good idea!

Reply:
Yes.